shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1356,7 +1456,7 @@ - (void)startKeyBoardAnimation:(NSTimeInterval)duration {
 
   // Set animation begin value.
   [self keyboardAnimationView].frame =
-      CGRectMake(0, _viewportMetrics.physical_view_inset_bottom, 0, 0);
+      CGRectMake(0, _viewportMetrics.physical_view_inset_bottom, 1, 1);
```

Comment:
Any reason for this change?

Reply:
just a remnant of some of my previous tests, will revert